### DISCUSSION TITLE
Own domain and certificates

### DISCUSSION THREAD

**Discussion by tsimur** [sherifabdlnaby/elastdocker#104] [Questions & Troubleshooting] [answered]
Hi, I like the concept of this repository and I'm going to use it to deploy the ELK stack. I followed the tutorial, but I noticed that the browser considers the certificates to be invalid. I thought I could use my domain and Let's Encrypt to make valid certificates. Could you share a step-by-step guide on how to do this in relation to running ELK using this repository.

**Comment by tsimur** [comment, marked as answer]
It seems I figured it out. After receiving certificates from let's encrypt, you need to put them in the `./secrets/certs/` folder in a subfolder of a specific service. For example, kibana (`./secrets/certs/kibana/`) instead of those certificates that were created there after the `make setup` command

**Comment by runxc1** [comment]
So you'd have to upload those every 3-4 months right?
